tester_app/client/assets/images

```diff
@@ -0,0 +1 @@
+/Users/Rob/dev/open-source/react_on_rails-generator-results/tester_app/app/assets/images
```

Comment:
I still can't figure out how to get these symlinks to be relative instead of absolute. This is actually possible, right?